src/atoms/buttons/transparent/index.astro

```diff
@@ -1,7 +1,7 @@
-<button class="border-2 border-sky-400 bg-transparent hover:bg-sky-400 px-12 md:px-20 py-4 md:py-6 text-lg font-medium text-white rounded-xl flex items-center justify-center gap-2">
+<button class="border-2 border-sky-400 bg-transparent hover:bg-sky-400 px-12 md:px-16 py-4 md:py-6 text-lg md:text-xl font-medium text-white rounded-xl flex items-center justify-center gap-2">
     <slot />
     <img
         class="size-6"
-        src="/images/button-arrow.svg"
+        src="/icons/button-arrow.svg"
```

Comment:
@Shivastoic The expectation is that the image would be a perfect square, if it is not then this size would make the image skewed since it would conflict with the image intrinsic size.